backends/eopkg/eopkgBackend.py

```
@@ -272,126 +281,165 @@ def _pkg_to_id(self, pkg):
+                candidates.append((available, INFO_AVAILABLE, repo))
+
+        for pkg, status, data in candidates:
+            if filters is not None:
+                if FILTER_GUI in filters and "app:gui" not in pkg.isA:
+                    continue
+                if FILTER_NOT_GUI in filters and "app:gui" in pkg.isA:
+                    continue
+                # FIXME: To lower
+                nonfree = ["EULA", "Distributable"]
+                if FILTER_FREE in filters:
+                    if any(l in pkg.license for l in nonfree):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)